tracer/src/Datadog.Trace/TracerManagerFactory.cs

```diff
@@ -112,12 +112,14 @@ internal TracerManager CreateTracerManager(ImmutableTracerSettings settings, Tra
                 runtimeMetrics ??= new RuntimeMetricsWriter(statsd ?? CreateDogStatsdClient(settings, defaultServiceName), TimeSpan.FromSeconds(10), settings.IsRunningInAzureAppService);
             }
 
+            var gitMetadataTagsProvider = new GitMetadataTagsProvider(settings);
```

Comment:
Can we do an abstraction for this, In CI Visibility we should use a different implementation by reading the values from: https://github.com/DataDog/dd-trace-dotnet/blob/master/tracer/src/Datadog.Trace/Ci/CIEnvironmentValues.cs

Reply:
Fixed in 22fac7f5ae73f8ee999a330998728232dca22389

Reply:
Thanks!